src/modules/cmdpal/Microsoft.CmdPal.UI/Settings/GeneralPage.xaml

```diff
@@ -34,20 +34,19 @@
                   <RepositionThemeTransition IsStaggeringEnabled="False" />
               </StackPanel.ChildrenTransitions>-->
                     <controls:SettingsExpander
-                        Description="This key will open the Command Palette"
-                        Header="Activation key"
+                        x:Uid="Settings_GeneralPage_ActivationKey_SettingsExpander"
                         HeaderIcon="{ui:FontIcon Glyph=&#xEDA7;}"
                         IsExpanded="True">
                         <ptControls:ShortcutControl HotkeySettings="{x:Bind viewModel.Hotkey, Mode=TwoWay}" />
 
                         <controls:SettingsExpander.Items>
-                            <controls:SettingsCard
-                                Description="If enabled, the Command Palette will return to the home page when activated"
-                                Header="Go home when activated"
-                                HeaderIcon="{ui:FontIcon Glyph=&#xE80F;}">
+                            <controls:SettingsCard x:Uid="Settings_GeneralPage_GoHome_SettingsCard" HeaderIcon="{ui:FontIcon Glyph=&#xE80F;}">
                                 <ToggleSwitch IsOn="{x:Bind viewModel.HotkeyGoesHome, Mode=TwoWay}" />
                             </controls:SettingsCard>
-                            <controls:SettingsCard Description="When enabled, the previous search text will be selected when the app is opened" Header="Highlight search on activate">
+                            <controls:SettingsCard
+                                x:Uid="Settings_GeneralPage_HighlightSearch_SettingsCard"
+                                Description=""
+                                Header="">
```

Comment:
Does the `Uid` win out over the empty strings here?

Reply:
Damn it. I'll fix it